flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/TestChangeLogTable.java

```diff
@@ -98,7 +98,7 @@ public void before() {
   @Override
   public void clean() {
     sql("DROP TABLE IF EXISTS %s", TABLE_NAME);
-    sql("DROP DATABASE IF EXISTS %s", DATABASE_NAME);
+    dropDatabase(DATABASE_NAME, true);
```

Comment:
Do we really need this? Something behind the scenes issue a `USE DATABASE` command?

If so, this could be a breaking change which we might want to highlight, as creating and dropping a database immediately after that won't work anymore

Reply:
After a discussion offline with @pvary, we have decided to create a new PR with a better cleaning up logic that will be applied to the unit tests. After that is merged, we can continue with this PR. 
Thanks!

Reply:
I ended up changing this implementation to:
```
  protected void dropDatabase(String database, boolean ifExists) {
    sql("USE CATALOG %s", DEFAULT_CATALOG_NAME);
    sql("USE %s", DEFAULT_DATABASE_NAME);
    sql("DROP DATABASE %s %s", ifExists ? "IF EXISTS" : "", database);
  }
```
which is similar to our current `dropCatalog` implementation.